forth/format.fs

```diff
@@ -1,11 +1,9 @@
-variable end
-create #>buf #34 allot
-: <# #>buf end ! ;
-: #> 2drop #>buf end @ over - ;
-: hold
-\ reserve space for char at start
-#>buf dup 1+ end @ #>buf - move
-1 end +! #>buf c! ;
+variable holdp
+#34 allot here dup
+: <# literal holdp ! ;
+: #> 2drop holdp @ literal over - ;
+: hold -1 holdp +! holdp @ c! ;
```

Comment:
Isn't there a risk for buffer overrun with this solution? When calling `HOLD` more than 34 times, `holdp` would move past beginning of buffer?

Reply:
...maybe that is fine, though? I think I am in favor of this solution.

Reply:
Same problem was in the original as well. After 34 holds it would start to overwrite the code for `<#`.